#1 is upstream in bdf641b5fda40, avoid installing bogus files under site-packages

Index: pyproject.toml
--- pyproject.toml.orig
+++ pyproject.toml
@@ -38,9 +38,10 @@ script-files = ["mat2"]
 
 [tool.setuptools.packages.find]
 exclude = ["tests*"]
+namespaces = false
 
 [tool.setuptools.data-files]
-"share/man/man1" = ["doc/mat2.1"]
+"man/man1" = ["doc/mat2.1"]
 
 [tool.ruff]
 target-version = "py38"
